### PR TITLE
Add new master link for C# sample apps

### PR DESCRIPTION
**Pull request by sgchoi5**
Signed-off-by: Sunggyu Choi <sunggyu.choi@samsung.com>

### CODE REVIEW

File: docs/application/dotnet/samples/tizen-samples.md [hunk 1]
```diff
@@ -101,8 +101,8 @@
 			</td>
 			<td>
 			<p><strong>FlexContainer Sample</strong></p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/tizen-samples.md]: We have duplicated "FlexContainer Sample" strings. We need to delete above one.

File: docs/application/dotnet/samples/tizen-samples.md [hunk 2]
```diff
@@ -254,6 +252,7 @@
 			</td>
 			<td>
 			<p><strong>FirstScreen</strong></p>
+                        <p>FirstScreen is a sample application that demonstrates how to launcher the apps on the TV.</p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/tizen-samples.md]: `how to launcher` into `how to launch`.

**Comment by mijin85cho** [comment]
@annie-samsung Hello, Annie. As this PR has not much updates, how about just checking the updated strings only and proceed to merge?

Regards

File: docs/application/dotnet/samples/tizen-samples.md (hunk 2 above)
Comment (annie-samsung) [inline docs/application/dotnet/samples/tizen-samples.md]: If FirstScreen sample app lets you to launch only one app on the TV, then please change the sentence to:

<p>FirstScreen is a sample application that demonstrates how to launch an app on the TV.</p>

**Comment by sgchoi5** [comment]
Probably, something wrong.. I will close this PR and will raise new one again....

**Pull request closed without merging**